compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2506,15 +2509,15 @@ impl Default for TargetOptions {
             pre_link_objects_self_contained: Default::default(),
             post_link_objects_self_contained: Default::default(),
             link_self_contained: LinkSelfContainedDefault::False,
-            pre_link_args: LinkArgs::new(),
+            pre_link_args: Default::default(),
```

Comment:
The default here still results in `Lazy`, rather than in empty `Owned`, from what I see.

Reply:
Ah, right, didn't notice.